### ISSUE TITLE
Wrap list and dictionary output

### ISSUE THREAD

**Issue by wasimsandhu** [marimo-team/marimo#2977]
### Description

The values for my dictionary keys are really long and it's awkward to scroll a string horizontally to read it.

### Suggested solution

Can this view be wrapped by default?

### Alternative

A toggle in the context menu to wrap outputs.

**Comment by mscolnick** [comment]
Without viewing long lists all that much, i feel like not wrapping as the default seems better. It is more often you are aren't look at all the values and i think it would clutter up the output area in every case, instead of just when you want to read it. Open to change my mind if you can share some examples.

**Comment by wasimsandhu** [comment]
I agree that it would make scanning all key-values harder if wrapped by default. A toggle in the right hand corner would be better then.

I can't share my actual data, but I am dealing with tables with 100+ columns, and each column has lots of text. This is what it looks like when I try to view a selected row as a dict:

![Image](https://github.com/user-attachments/assets/384e0145-d4b5-4901-8ca0-107c70483c43)

It would be really helpful if it was wrapped, similar to what viewing a large JSON document in a web page looks like.

**Comment by mscolnick** [comment]
Yea, this can be hard to parse, maybe we can support different views. You can check out https://jsonhero.io/ for some inspiration

**Comment by wasimsandhu** [comment]
I was thinking a "Wrap text" toggle on the right hand corner would suffice for my use case. Can open a PR later this week if so, otherwise we can scope it out some more
